lib/rules/shorthand-property-no-redundant-values/index.mjs

```diff
@@ -39,6 +40,21 @@ const SUPPORTED_SHORTHANDS = new Set([
 	'inset', 'inset-block', 'inset-inline',
 ]);
 
+// prettier-ignore
+const EDGE_PROPERTIES = new Set([
+	'margin', 'margin-block', 'margin-inline',
+	'padding', 'padding-block', 'padding-inline',
+	'border-color', 'border-style', 'border-width',
+	'border-block-color', 'border-block-style', 'border-block-width',
+	'border-inline-color', 'border-inline-style', 'border-inline-width',
+	'gap', 'grid-gap',
+	'overflow',
+	'overscroll-behavior',
+	'scroll-margin', 'scroll-margin-block', 'scroll-margin-inline',
+	'scroll-padding', 'scroll-padding-block', 'scroll-padding-inline',
+	'inset', 'inset-block', 'inset-inline',
+]);
```

Comment:
nit

Since we don't have `Set.prototype.intersection`, you could refactor this using our util.

```js
// prettier-ignore
const EDGE_PROPERTIES = new Set([
	'margin', 'margin-block', 'margin-inline',
	'padding', 'padding-block', 'padding-inline',
	'border-color', 'border-style', 'border-width',
	'border-block-color', 'border-block-style', 'border-block-width',
	'border-inline-color', 'border-inline-style', 'border-inline-width',
	'gap', 'grid-gap',
	'overflow',
	'overscroll-behavior',
	'scroll-margin', 'scroll-margin-block', 'scroll-margin-inline',
	'scroll-padding', 'scroll-padding-block', 'scroll-padding-inline',
	'inset', 'inset-block', 'inset-inline',
]);

const SUPPORTED_SHORTHANDS = uniteSets(
	EDGE_PROPERTIES,
	['border-radius'],
);
```